web/setup.py

```diff
@@ -47,7 +47,8 @@
 from typing_extensions import Annotated
 from pgadmin.utils.constants import INTERNAL, LDAP, OAUTH2, \
     KERBEROS, WEBSERVER
-from pgadmin.tools.user_management import create_user, delete_user, update_user
+from pgadmin.tools.user_management import create_user, delete_user, \
+    update_user as update_user_internal
```

Comment:
The update_user will be used to update both internal and external auth users. So, please change the update_user_internal alias or leave it as the default update_user.

Reply:
OK, changed -- is `user_management_update_user` better? I'm just trying to avoid a function here called `update_user` calling another function called `update_user`